### PR TITLE
doc: Fix broken links referring to https://etcd.readthedocs.io/

### PR DESCRIPTION
**Pull request by retroflexer**
See the issue created here:
https://github.com/etcd-io/etcd/issues/10989#issuecomment-518726038

Please read https://github.com/etcd-io/etcd/blob/master/CONTRIBUTING.md#contribution-flow.

### CODE REVIEW

File: Documentation/op-guide/runtime-configuration.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ If adding multiple members the best practice is to configure a single member at
 #### Add a new member as learner
 
 Starting from v3.4, etcd supports adding a new member as learner / non-voting member.
-The motivation and design can be found in [design doc](https://etcd.readthedocs.io/en/latest/server-learner.html).
+The motivation and design can be found in [design doc](https://etcd.io/docs/v3.3.12/learning/learner/#raft-learner).
```
Comment (hexfusion) [inline Documentation/op-guide/runtime-configuration.md]: This document is part of the repo, the link should be relative to the repo vs a hardcoded version. See examples here: https://raw.githubusercontent.com/etcd-io/etcd/master/Documentation/op-guide/clustering.md

Comment (spzala) [reply Documentation/op-guide/runtime-configuration.md]: @hexfusion thanks, that sounds good. In the related issue https://github.com/etcd-io/etcd/issues/10989 we had a brief discussion and I think based on that @retroflexer used the etcd.io one. @retroflexer please use the other link in the format suggested by @hexfusion . Thanks!

**Comment by spzala** [comment]
@retroflexer there is one more broken link related to the readthedocs - https://github.com/etcd-io/etcd/blob/master/CHANGELOG-3.3.md#metrics-monitoring (List of metrics). As part of this PR can you please replace it with https://etcd.io/docs/v3.3.12/metrics/ ? Thanks! 
/cc @gyuho @hexfusion

**Comment by retroflexer** [comment]
> @retroflexer there is one more broken link related to the readthedocs - https://github.com/etcd-io/etcd/blob/master/CHANGELOG-3.3.md#metrics-monitoring (List of metrics). As part of this PR can you please replace it with https://etcd.io/docs/v3.3.12/metrics/ ? Thanks!
> /cc @gyuho @hexfusion

@spzala Looks like it is already pointing to etcd.io/docs. Do you instead want it to refer to the page internal to this repo?

**Pull request closed without merging**